src/box/relay.cc

```diff
@@ -179,6 +183,8 @@ struct relay {
 	double txn_lag;
 	/** Relay sync state. */
 	enum relay_state state;
+	/** Whether relay should speed up the next heartbeat dispatch. */
+	bool speedup_heartbeat;
```

Comment:
"Speedup" is a verb, which makes this name look like an action, a function. I would recommend to start all flags with `is/has/does/need/...` to emphasize this is just a flag. For example, here it could be `do_bump_heartbeat` or `need_new_heartbeat` or likewise.

Reply:
Let it be `need_new_heartbeat`. Fixed.